src/app/services/code-view/angular-code-service.ts

```diff
@@ -372,4 +375,23 @@ export class AngularCodeService extends CodeService {
         fileInput.appendTo(form)
         return form;
     }
-}
\ No newline at end of file
+
+    private checkForDvSample(demoFileMetadataName: string, demosBaseUrl: string): string {
```

Comment:
This method actually mutates the parameters and returns a new modified string. However, the method name does not tell us what this method does. Use better naming to properly express the method goal.